[iOS] Failed to build SwiftGen

**Overview (Required)**
- Failed to build SwiftGen.

**Steps To Reproduce**

1. Go to `ios_base` directory.
2. Run `make init`
3. `Failed to build SwiftGen 6.1.0 with SPM`

**Expected behavior**
the build to be success.
 
**Env:**
 - OS: MacOS 10.15.3
- Xcode 11.3.1
 - master branch / 2bedc70e496d3abc207754cd2a178e96b626d86e

**Error Log**

```
mint run swiftgen swiftgen
🌱  Using SwiftGen/SwiftGen 6.1.0 from Mintfile.
🌱  Cloning SwiftGen 6.1.0
🌱  Resolving package
🌱  Building package
[1/10] Compiling writer.c
[2/10] Compiling scanner.c
[3/10] Compiling reader.c
[4/10] Compiling parser.c
[5/10] Compiling emitter.c
[6/10] Compiling api.c
[7/11] Compiling Kanna CSS.swift
/private/var/folders/r9/d3j1xpps38q2h51974d_hvgw0000gn/T/mint/github.com_SwiftGen_SwiftGen/.build/checkouts/Kanna/Modules/libxml2-kanna.h:1:9: note: while building module 'libxml2' imported from /private/var/folders/r9/d3j1xpps38q2h51974d_hvgw0000gn/T/mint/github.com_SwiftGen_SwiftGen/.build/checkouts/Kanna/Modules/libxml2-kanna.h:1:
#import <libxml2/libxml/HTMLtree.h>
        ^
<module-includes>:1:9: note: in file included from <module-includes>:1:
#import "libxml/HTMLparser.h"
        ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/usr/include/libxml2/libxml/HTMLparser.h:15:10: error: 'libxml/xmlversion.h' file not found
#include <libxml/xmlversion.h>
         ^
<module-includes>:1:9: note: in file included from <module-includes>:1:
#import "libxml2-kanna.h"
        ^
/private/var/folders/r9/d3j1xpps38q2h51974d_hvgw0000gn/T/mint/github.com_SwiftGen_SwiftGen/.build/checkouts/Kanna/Modules/libxml2-kanna.h:1:9: error: could not build module 'libxml2'
#import <libxml2/libxml/HTMLtree.h>
        ^
/private/var/folders/r9/d3j1xpps38q2h51974d_hvgw0000gn/T/mint/github.com_SwiftGen_SwiftGen/.build/checkouts/Kanna/Sources/Kanna/CSS.swift:27:8: error: could not build Objective-C module 'libxmlKanna'
import libxmlKanna
       ^
'SwiftClibxml2' /private/var/folders/r9/d3j1xpps38q2h51974d_hvgw0000gn/T/mint/github.com_SwiftGen_SwiftGen/.build/checkouts/SwiftClibxml2: warning: system packages are deprecated; use system library targets instead
🌱  Encountered error during "swift build -c release -Xswiftc -target -Xswiftc x86_64-apple-macosx10.15". Use --verbose to see full output
🌱  Failed to build SwiftGen 6.1.0 with SPM
make: *** [swiftgen] Error 1
```

--- a/ios-base/Makefile
+++ b/ios-base/Makefile
@@ -1,6 +1,3 @@
-GIT_REPO = $(shell git rev-parse --show-toplevel)
-GIT_DIR = $(shell git rev-parse --git-dir)
-
 init: 
 	scripts/bootstrap.sh
 
@@ -9,7 +6,7 @@ xcode: swiftgen
 	bundle exec pod install
 
 swiftgen:
-	mkdir -p $(GIT_REPO)/ios-base/DroidKaigi\ 2020/Generated
+	mkdir -p "DroidKaigi 2020/Generated"
 	mint run swiftgen swiftgen
 
 test: xcode

--- a/ios-base/Makefile
+++ b/ios-base/Makefile
@@ -1,5 +1,5 @@
 init: 
-	scripts/bootstrap.sh
+	@scripts/bootstrap.sh
 
 xcode: swiftgen
 	mint run xcodegen

--- a/ios-base/Makefile
+++ b/ios-base/Makefile
@@ -1,3 +1,6 @@
+.PHONY: default
+default: xcode
+
 init: 
 	@scripts/bootstrap.sh
 
@@ -6,7 +9,7 @@ xcode: swiftgen
 	bundle exec pod install
 
 swiftgen:
-	mkdir -p "DroidKaigi 2020/Generated"
+	@mkdir -p "DroidKaigi 2020/Generated"
 	mint run swiftgen swiftgen
 
 test: xcode

--- a/ios-base/Makefile
+++ b/ios-base/Makefile
@@ -1,11 +1,8 @@
 GIT_REPO = $(shell git rev-parse --show-toplevel)
 GIT_DIR = $(shell git rev-parse --git-dir)
 
-init: swiftgen
-	mint run xcodegen
-	bundle install --path vendor/bundle
-	bundle exec pod install
-	@cp $(GIT_REPO)/ios-base/scripts/git-hooks/* $(GIT_DIR)/hooks
+init: 
+	scripts/bootstrap.sh
 
 xcode:
 	mint run xcodegen

--- a/ios-base/Makefile
+++ b/ios-base/Makefile
@@ -4,7 +4,7 @@ GIT_DIR = $(shell git rev-parse --git-dir)
 init: 
 	scripts/bootstrap.sh
 
-xcode:
+xcode: swiftgen
 	mint run xcodegen
 	bundle exec pod install
 

--- a/ios-base/scripts/bootstrap.sh
+++ b/ios-base/scripts/bootstrap.sh
@@ -1,0 +1,58 @@
+#!/bin/zsh
+
+GIT_REPO=$(git rev-parse --show-toplevel)
+GIT_DIR=$(git rev-parse --git-dir)
+
+# Install Homebrew
+if ! type "brew" > /dev/null; then
+    echo '`brew` not found. Install Homebrew'
+    ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+fi
+
+# rbenv
+if ! type "rbenv" > /dev/null; then
+    echo '`rbenv` not found. Install rbenv'
+    brew install rbenv rbenv-communal-gems
+
+    cat << EOS >> ~/.zshrc
+    if which rbenv > /dev/null; then eval "\$(rbenv init -)"; fi
+    export PATH="\$HOME/.rbenv/bin:\$PATH"
+    eval "\$(rbenv init -)"
+EOS
+fi
+
+source ~/.zshrc
+
+# Ruby
+RUBY_VERSION="$(rbenv local)"
+if [ "$(rbenv versions --bare | grep ${RUBY_VERSION})" = '' ]; then
+    rbenv install ${RUBY_VERSION}
+fi
+
+# Dependency for Swiftgen
+echo 'Install libxml2'
+brew install libxml2
+
+# Install Mint dependencies
+if ! type "mint" > /dev/null; then
+    echo '`mint` not found. Install Mint'
+    git clone https://github.com/yonaskolb/Mint.git
+    make -C Mint
+    rm -rf Mint
+fi
+
+mint bootstrap
+
+# Swiftgen
+mkdir -p "${GIT_REPO}/ios-base/DroidKaigi 2020/Generated"
+mint run swiftgen swiftgen
+
+# XcodeGen
+mint run xcodegen
+
+# Bundle(Gems)
+bundle install
+bundle exec pod install
+
+# Git-hooks
+cp ${GIT_REPO}/ios-base/scripts/git-hooks/* ${GIT_DIR}/hooks

--- a/ios-base/scripts/bootstrap.sh
+++ b/ios-base/scripts/bootstrap.sh
@@ -3,6 +3,15 @@
 GIT_REPO=$(git rev-parse --show-toplevel)
 GIT_DIR=$(git rev-parse --git-dir)
 
+if [ "$SHELL" = '/bin/bash' ]; then
+    SHELL_RESOURCE=~/.bashrc
+elif [ "$SHELL" = '/bin/zsh' ]; then
+    SHELL_RESOURCE=~/.zshrc
+else
+    echo "Unsupported shell: ${SHELL}"
+    exit 1
+fi
+
 # Install Homebrew
 if ! type "brew" > /dev/null; then
     echo '`brew` not found. Install Homebrew'
@@ -14,14 +23,14 @@ if ! type "rbenv" > /dev/null; then
     echo '`rbenv` not found. Install rbenv'
     brew install rbenv rbenv-communal-gems
 
-    cat << EOS >> ~/.zshrc
+    cat << EOS >> ${SHELL_RESOURCE}
     if which rbenv > /dev/null; then eval "\$(rbenv init -)"; fi
     export PATH="\$HOME/.rbenv/bin:\$PATH"
     eval "\$(rbenv init -)"
 EOS
 fi
 
-source ~/.zshrc
+source ${SHELL_RESOURCE}
 
 # Ruby
 RUBY_VERSION="$(rbenv local)"

